src/o365/spo/commands/mail/mail-send.spec.ts

```diff
@@ -127,9 +128,8 @@ describe(commands.MAIL_SEND, () => {
     auth.site.connected = true;
     auth.site.url = 'https://contoso.sharepoint.com';
     cmdInstance.action = command.action();
-    cmdInstance.action({ options: { debug: true, to: 'user@contoso.com', subject: 'Subject of the email', body: 'Content of the email', verbose: true } }, () => {
+    cmdInstance.action({ options: { debug: truncate, to: 'user@contoso.com', subject: 'Subject of the email', body: 'Content of the email' } }, () => {
```

Comment:
Is this intentional `debug: truncate`?

Reply:
It was... Not need anymore now.
Totally forgot to remove it :/